tests/fixtures/config/hooks/plugin_with_ui.py

```diff
@@ -372,7 +373,8 @@ def publish(self, settings, item):
             instances.
         :param item: Item to process
         """
-        self.logger.info("Plugin with UI data was published!")
+        print item, "was published! The settings were:"
```

Comment:
Should we log a `pprint`'d string?

Reply:
Logging a pprint'ed string shows up as a single line inside the publisher unfortunately.
This is for testing also, so the easier it is to validate the result the better it is I feel.